frontend/src/state/StudentActions.js

```diff
@@ -0,0 +1,38 @@
+import fetch from 'isomorphic-fetch';
+import { createAction } from 'utils/actionUtils';
+import * as types from './StudentConstants';
+// import { authRequest } from './UserActions';
+// import { requestStudents } from '../services/studentServices';
+
+export const setLoading = createAction(types.SET_LOADING);
+
+const getStudents = students => ({
+  type: types.SET_STUDENTS,
+  payload: students,
+});
+
+const getStudent = student => ({
+  type: types.SET_STUDENT,
+  payload: student,
+});
+
+export const fetchStudents = (students) => {
+  return (dispatch) => {
+    return fetch('http://gsndev.com/gsndb/student/', {
```

Comment:
We should have ROOT_URL constnat that holds 'http://gsndev.com/'

Reply:
Can you explain the reason that fetchStudents works and why the syntax is the way it is?